File: 0026_remove_duplicates_from_sorted_array/Solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int lastUniqueNum = nums[0], z = 1;

        for (int x : nums) {
            if (x != lastUniqueNum) {
                nums[z] = x;
                lastUniqueNum = nums[z];
                z++;
            }
        }

        return z;
    }
}

File: 0026_remove_duplicates_from_sorted_array/Solution.js
var removeDuplicates = function (nums) {
    let lastUnique = nums[0];
    let i = 1;

    for (let n of nums) {
        if (n != lastUnique) {
            nums[i] = n;
            lastUnique = n;
            i++;
        }
    }

    return i;
};
